spa/src/app.tsx

```diff
@@ -47,6 +48,7 @@ const App = () => {
 						<Route path="error" element={<ErrorState />} />
 					</Route>
 					<Route path="deferred" element={<DeferredInstallation />}/>
+					<Route path="repositories" element={<Repositories />}/>
```

Comment:
This route is not used anywhere at the moment. So please either remove this or add a TODO saying this needs to revisited in the future.

Reply:
Removed both route and page